ommx_fixstars_amplify_adapter/adapter.py

```diff
@@ -0,0 +1,289 @@
+import amplify
+
+from typing import Union
+
+from ommx.v1 import (
+    Solution,
+    Instance,
+    DecisionVariable,
+    Constraint,
+    State,
+)
+from ommx.v1.function_pb2 import Function
+from ommx.adapter import SolverAdapter
+from ommx.v1.linear_pb2 import Linear
+from ommx.v1.quadratic_pb2 import Quadratic
+from ommx.v1.polynomial_pb2 import Polynomial
+from ommx.v1.constraint_pb2 import Constraint as RawConstraint
+from ommx.v1.decision_variables_pb2 import DecisionVariable as RawDVar
+
+from .exception import OMMXFixstarsAmplifyAdapterError
+
+
+class OMMXFixstarsAmplifyAdapter(SolverAdapter):
+    def __init__(self, ommx_instance: Instance):
+        """
+        :param ommx_instance: The ommx.v1.Instance to solve.
+        """
+        self.instance = ommx_instance
+        self.model = amplify.Model()
+
+        self._set_decision_variables()
+        self._set_objective()
+        self._set_constraints()
+
+    @staticmethod
+    def solve(
+        ommx_instance: Instance, amplify_token: str = "", timeout: int = 1000
```

Comment:
```suggestion
        ommx_instance: Instance, *, amplify_token: str = "", timeout: int = 1000
```
According to [the guide](https://jij-inc.github.io/ommx/ja/user_guide/solver_adapter.html), arguments other than `ommx_instance` must be keyword-only.

Reply:
This made me realize we forgot to add the `*` to the [python-mip adapter](https://github.com/Jij-Inc/ommx/blob/main/python/ommx-python-mip-adapter/ommx_python_mip_adapter/adapter.py#L64)